commit bdedd43ce0dc56d8fededdbe1e4cde1cf2e4f6be (refs/changes/55/1055/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-05T10:09:02-08:00 (2 years, 10 months ago)
    
    TOOLS-1628 sdcadm agent install should work for cmon-agent
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/etc/defaults.json b/etc/defaults.json
index dda9277..b4a508e 100644
--- a/etc/defaults.json
+++ b/etc/defaults.json
@@ -39,6 +39,7 @@
         "cainstsvc": "cainstsvc",
         "amon-relay": "amon-relay",
         "smartlogin": "smartlogin",
+        "cmon-agent": "cmon-agent",
         "cn-agent": "cn-agent",
         "vm-agent": "vm-agent",
         "net-agent": "net-agent",
diff --git a/lib/procedures/index.js b/lib/procedures/index.js
index 79ce4e3..17131d2 100644
--- a/lib/procedures/index.js
+++ b/lib/procedures/index.js
@@ -688,7 +688,7 @@ function coordinatePlan(opts, cb) {
             var allowed = ['cn-agent', 'vm-agent', 'net-agent',
                 'agents_core', 'firewaller', 'smartlogin', 'config-agent',
                 'amon-agent', 'amon-relay', 'hagfish-watcher', 'cabase',
-                'cainstsvc'
+                'cainstsvc', 'cmon-agent'
             ];
             // Attempts on updating the following agents using this method will
             // result into an error message:
